app/models/shipit/undeployed_commit.rb

```diff
@@ -11,15 +11,14 @@ def deploy_state(bypass_safeties = false)
       state = deployable? ? 'allowed' : status.state
       unless bypass_safeties
         state = 'deploying' if stack.active_task?
-        state = 'locked' if stack.locked?
+        state = 'locked' if locked?
```

Comment:
Good question.

I think `Locked` was always a bit confusing to people. So I though, might as well make the stack level lock not change the button caption, since it's obvious enough already.